qiskit_ibm_experiment/client/local_client.py

```diff
@@ -268,51 +269,58 @@ def experiments(
-                    df.tags.apply(lambda dftags: all(x in dftags for x in tags))
+                    df.tags.apply(lambda dftags: all(x in dftags for x in tags_list))
                 ]
             else:
                 raise ValueError("Unrecognized tags operator")
 
-        if "start_datetime_before" in filters:
-            df = df.loc[df.start_time <= filters["start_datetime_before"]]
-        if "start_datetime_after" in filters:
-            df = df.loc[df.start_time >= filters["start_datetime_after"]]
+        start_datetime_before = None
+        start_datetime_after = None
+        if "start_time" in filters:
+            for start_time_string in filters["start_time"]:
+                string_type = start_time_string[:2]
+                value = str_to_utc(start_time_string[3:])
+                if string_type == "ge":
+                    start_datetime_after = value
+                if string_type == "le":
+                    start_datetime_before = value
+
+        if start_datetime_before is not None:
+            df = df.loc[
+                lambda df: df.start_time.apply(str_to_utc) <= start_datetime_before
+            ]
+        if start_datetime_after is not None:
+            df = df.loc[
+                lambda df: df.start_time.apply(str_to_utc) >= start_datetime_after
+            ]
 
         sort_by = filters.get("sort_by")
         if sort_by is None:
-            sort_by = "start_datetime:desc"
-
-        if not isinstance(sort_by, list):
-            sort_by = [sort_by]
-
-        # TODO: support also experiment_type
-        if len(sort_by) != 1:
-            raise ValueError(
-                "The fake service currently supports only sorting by start_datetime"
-            )
-
-        sortby_split = sort_by[0].split(":")
-        # TODO: support also experiment_type
-        if (
-            len(sortby_split) != 2
-            or sortby_split[0] != "start_datetime"
-            or (sortby_split[1] != "asc" and sortby_split[1] != "desc")
-        ):
-            raise ValueError(
-                "The fake service currently supports only sorting by start_datetime, which can be "
-                "either asc or desc"
-            )
-
-        df = df.sort_values(
-            ["start_time", "uuid"], ascending=[(sortby_split[1] == "asc"), True]
-        )
-
+            sort_by = "start_time:desc"
+        sort_by = sort_by.split(",")
+
+        sort_by_columns = []
+        sort_by_ascending = []
+        for sort_by_element in sort_by:
+            sortby_split = sort_by_element.split(":")
+            if len(sortby_split) != 2 or (
+                sortby_split[1] != "asc" and sortby_split[1] != "desc"
+            ):
+                raise ValueError(f"Sortby filter {sort_by} is malformed")
+            sort_by_columns.append(sortby_split[0])
+            sort_by_ascending.append(sortby_split[1] == "asc")
+
+        sort_by_columns.append("uuid")
+        sort_by_ascending.append(True)
```

Comment:
Alternatively before line 308 you could write
```
sort_by += ",uuid:asc"
```
I find it a bit more readable because you treat `start_time` and `uuid` in the same way